api/app/Exceptions/ErrorCode.php

```diff
@@ -18,5 +18,6 @@ final class ErrorCode
     public const WEEKEND_EXCEPTION = 1009;
     public const USER_ALREADY_VERIFIED_EXCEPTION = 1010;
     public const EMAIL_IS_NOT_VERIFIED_EXCEPTION = 1011;
-    public const WRONG_DATE_TIME_EXCEPTION = 1012;
```

Comment:
Do not change existing codes. They might be already used on the client.

Reply:
Yep sorry, did the merge in the late of night mix it up